tasks/main.yml

```diff
@@ -79,6 +79,23 @@
   when: cockpit_config is defined
   notify: restart cockpit
 
+- name: Create certificates
+  when:
+    - cockpit_certificates | length > 0
+    - ansible_facts['os_family'] == 'RedHat'
```

Comment:
The node in the readme says `NOTE: This does *not* work on RHEL/CentOS 7.` - do we need to add a condition for that e.g.
```yaml
    - ansible_facts['distribution_version'] is version('>=', '8')
```
?

Reply:
Ahh, sorry. The note is not needed. The issue was if we try to generate the new cert directly in the target location `/etc/cockpit/ws-certs.d` like this:
```
    cockpit_certificates:
      - name: /etc/cockpit/ws-certs.d/cockpit_cert
        <<snip>>
```
it fails due to [this bug](https://github.com/linux-system-roles/certificate/pull/97) on RHEL/CentOS-7 with an invalid mode.

In this PR, it generates the private key and cert in the standard location and copies them to the directories Cockpit expects. So, it is free from the issue. And there's no problem running on RHEL/CentOS-7. (CI successfully passed on the platforms.) I'm removing the note. Thanks for finding it out, @richm!

Reply:
> Do we want to describe in a little more detailed manner something like this?

Yes - and - do we have a bz or some other issue tracker for this issue?